### PR TITLE
Add pagination first last pages

### PR DESCRIPTION
**Pull request by ykyuen**
I am looking for adding first and last page in the pagination component and i found the following pull request.
https://github.com/couds/react-bulma-components/pull/100

The code works great so i forked the latest master and open this new pull request.
Thanks @haflinger and all the authors who have been working on this great module.

### CODE REVIEW

**Comment by haflinger** [comment]
Thanks @ykyuen !

**Comment by ykyuen** [comment]
In addition to the first and last pages, **rounded**, **size** and **position** are also added in pagination.

Usage:
```
<Pagination
  ...
  size={ 'small' }
  rounded={ true }
  position={ 'right' }
  ...
/>
```

**Comment by MrCreeper1008** [comment]
@ykyuen thanks for the pr. can you merge into `next` branch instead?

File: src/components/pagination/pagination.js [hunk 1]
```diff
@@ -95,6 +103,25 @@ export default class Pagination extends React.PureComponent {
...
           <>
             <ul className="pagination-list">
+              {showFirstLast && current !== 1 && firstPage !== 1 && (
+                <>
+                  <li key={1}>
+                    <a
+                      role="button"
+                      tabIndex={0}
+                      className="pagination-link"
+                      onClick={current === 1 ? undefined : this.goToPage(1)}
```
Comment (couds) [inline src/components/pagination/pagination.js:113]: I think we can remove this `current === 1` . you already check this condition on line 106

**Review by couds — CHANGES_REQUESTED** [review]
LGTM, just one small change

File: src/components/pagination/pagination.js (hunk 1 above)
Comment (ykyuen) [reply src/components/pagination/pagination.js]: Thanks for pointing that out. i have made another commit to remove the duplicated checking on both first and last buttons.

**Review by MrCreeper1008 — CHANGES_REQUESTED** [review]
Looks good, can you also update the typescript def as well? Thank you!

**Comment by ykyuen** [comment]
Hi @MrCreeper1008, i am not familiar with typescript. please let me know if it works or not.

**Review by MrCreeper1008 — CHANGES_REQUESTED** [review]
Almost there, but I think you forgot to mark `position` and `rounded` as optional.

**Comment by ykyuen** [comment]
@MrCreeper1008 Thanks for letting me know and sorry for the miss. i have updated the code.

**Review by MrCreeper1008 — APPROVED** [review]
Looks good!

**Pull request merged**